src/lib.rs

```diff
@@ -85,22 +88,7 @@ pub struct Message {
     pub sig: Option<Signature>,
 }
 
-impl Message {
-    /// Returns a new Message instance
-    pub fn new(data: Vec<u8>) -> Message {
-        Message {
-            msg: Box::new(data),
-            d: None,
-            sym_nonce: None,
-            asym_nonce: None,
-            digest: Ok(vec![]),
-            op_result: Ok(()),
-            sig: None,
-        }
-    }
-}
```

Comment:
I would just say lets keep this here since ```Message``` is declared here, see other comments. ```ops.rs``` is the file where all of the cryptographic operations are defined, this sort of falls outside of that and is more of a utility function, so are the file reading and writing functions